GoogleForCreators/web-stories-wp
# Media: Generate and backfill BlurHash for media items

## Context



## Summary

Version 2 of this PR.  Version 1 https://github.com/google/web-stories-wp/pull/9935

Generate blurhashs for all media and existing media elements using the blurhash library. As the blurhash is slow process, use a web worker, to do the calculation in another thread.  

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a new story. 
2. Upload a video or image. 
3. Wait until upload is complete. 
4. Wait for 20 seconds. 
5. Reload the editor. 
6. See blurhash for newly uploaded asset. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9685
